addons/im_livechat/static/src/embed/common/chatbot/chatbot_model.js

```diff
@@ -13,6 +13,13 @@ export class Chatbot extends Record {
     static MULTILINE_STEP_DEBOUNCE_DELAY_TOUR = 500;
 
     isTyping = false;
+    /**
+     * Indicate that the chat bot is processing a user answer. It's the case as
+     * soon as the user sends a message and until the server answer is fully
+     * inserted into the store. Used to disable the composer during the
+     * processing.
+     */
+    isProcessingAnswer = false;
```

Comment:
Comment is redundant with its use in code that is trivial, I would remove as to keep code short and simple, and avoid maintaining a comment that could easily become a lie in the future.